gateways/authorizenet/authorizenet.go

```diff
@@ -129,6 +129,16 @@ func (client *AuthorizeNetClient) Refund(request *sleet.RefundRequest) (*sleet.R
 
 // RefundWithContext refunds a captured transaction with amount and captured transaction reference
 func (client *AuthorizeNetClient) RefundWithContext(ctx context.Context, request *sleet.RefundRequest) (*sleet.RefundResponse, error) {
+	transactionDetailsResponse, err := client.GetTransactionDetails(&sleet.TransactionDetailsRequest{
```

Comment:
Curious: so is this only used for Google Pay? Maybe we could skip this call for non-GooglePay request?